lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -0,0 +1,796 @@
+  oss << String( awidth , '-' ) << "\n";
+  st = "Multiple R-squared";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << test1;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n";
+  st = "Adjusted R-squared";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << test2;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n" << String( awidth , '-' ) << "\n";
+
+  // normality tests
+  lwidth = 0;
+  twidth = 20;
+  const Scalar normalitytest1(getNormalityTestResultAndersonDarling().getPValue());
+  const Scalar normalitytest2(getNormalityTestResultChiSquared().getPValue());
+  const Scalar normalitytest3(getNormalityTestResultKolmogorovSmirnov().getPValue());
+  const Scalar normalitytest4(getNormalityTestCramerVonMises().getPValue());
+  st = OSS() << normalitytest1;
+  lwidth = std::max( lwidth, st.size() );
+  st = OSS() << normalitytest2;
+  lwidth = std::max( lwidth, st.size() );
+  st = OSS() << normalitytest3;
+  lwidth = std::max( lwidth, st.size() );
+  st = OSS() << normalitytest4;
+  lwidth = std::max( lwidth, st.size() );
+  awidth = twidth + 2 * separator.size() + lwidth - 1;
+  oss << "\n" <<  String( awidth , '-' ) << "\n";
+  st = "Normality test";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = "p-value";
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n" <<  String( awidth , '-' ) << "\n";
+  st = "Anderson-Darling";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << normalitytest1;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n";
+  st = "Cramer-Von Mises";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << normalitytest4;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n";
+  st = "Chi-Squared";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << normalitytest2;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n";
+  st = "Kolmogorov-Smirnov";
+  oss << st << String( twidth - st.size(), ' ') << separator;
+  st = OSS() << normalitytest3;
+  oss << st << String( lwidth - st.size(), ' ') << separatorEndLine;
+  oss << "\n" <<  String( awidth , '-' ) << "\n";
+  return oss;
+}
+
+
+/* Linear model accessor */
+LinearModelResult LinearModelAnalysis::getLinearModelResult() const
+{
+  return linearModelResult_;
+}
+
+String LinearModelAnalysis::getFormula() const
+{
+  return linearModelResult_.getFormula();
+}
+
+Description LinearModelAnalysis::getCoefficientsNames() const
+{
+  return linearModelResult_.getCoefficientsNames();
+}
+
+Sample LinearModelAnalysis::getResiduals() const
+{
+  return linearModelResult_.getSampleResiduals();
+}
+
+Sample LinearModelAnalysis::getStandardizedResiduals() const
+{
+  return linearModelResult_.getStandardizedResiduals();
```

Comment:
I dont see the point of theses methods that just returns attributes of the result class

Reply:
You are probably right, but I prefer integrating class as implemented in otlm
In a second step, I will update them (add methods to result, make analysis class too specific...)

Reply:
ok fixed. I removed all similar methods also